deployments/rialto/docker-compose.yml

```diff
@@ -148,16 +148,34 @@ services:
   prometheus-metrics:
     image: prom/prometheus
     volumes:
-      - ../dashboard/prometheus/:/etc/prometheus/
+      - ./dashboard/prometheus/:/etc/prometheus/
     ports:
       - "9090:9090"
     depends_on: *all-nodes
 
   grafana-dashboard:
     image: grafana/grafana
+    environment:
+      VIRTUAL_HOST: dashboard.rialto.bridges.test-installations.parity.io,grafana.rialto.brucke.link
+      VIRTUAL_PORT: 3000
+      LETSENCRYPT_HOST: dashboard.rialto.bridges.test-installations.parity.io,grafana.rialto.brucke.link
+      LETSENCRYPT_EMAIL: admin@parity.io
+      GF_SECURITY_ADMIN_PASSWORD: ${GRAFANA_ADMIN_PASS:-admin}
+      GF_SERVER_ROOT_URL: ${GRAFANA_SERVER_ROOT_URL}
     volumes:
-      - ../dashboard/grafana/provisioning/:/etc/grafana/provisioning/
+      - ./dashboard/grafana/provisioning/:/etc/grafana/provisioning/
     ports:
       - "3000:3000"
     depends_on:
       - prometheus-metrics
+
+  grafana-matrix-notifier:
```

Comment:
Is there some way to make this an optional step? If I want to test a network locally I don't really want to have this

Reply:
The container will just fail to start or will simply not send notifications. AFAIK there is no way to have conditional services in compose file.